src/app/home/tabs/post/data-access/post-state.service.ts

```diff
@@ -44,16 +47,21 @@ export class PostStateService {
 		)
 	);
 
+	userLocation = from(Geolocation.getCurrentPosition());
```

Comment:
This does not always work for me, If I refresh and take a foto for the very first time there Is a blue marker with my current location (awesome!) but if I cancel and retake one the marker does not appear for any of the following pictures. If I cancel by refreshing it works (or just refresh again) probably something wrong with the cancel button

Reply:
d0d781c5631af8692a23b9e97ae4f3501a38d32f